minkindr_gtsam/src/quat-transformation-gtsam.cc

```diff
@@ -196,9 +196,9 @@ Eigen::Vector3d rotationFromTransformationLogImplementation(const kindr::minimal
   }
 }
 
-Expression<Eigen::Vector3d> rotationLog(
-    const Expression<kindr::minimal::QuatTransformation>& T) {
-  return Expression<Eigen::Vector3d>(&rotationFromTransformationLogImplementation, T);
+EVector3 rotationLog(
+    const ETransformation& T) {
+  return EVector3(&rotationFromTransformationLogImplementation, T);
 }
 
 kindr::minimal::QuatTransformation invertAndComposeImplementation(
```

Comment:
@furgalep Would it be desirable to have some of these implementations accessible publicly, or only a slerp implementation? It would enable us to evaluate the curve when we are not interested about jacobians and wish not to provide gtsam values for efficiency sake. Currently, these implementations are mirrored in curves. What do you think? Thanks!